### ISSUE TITLE
[Firestore] Timestamp.now().seconds broken

### ISSUE THREAD

**Issue by tfang17** [firebase/firebase-admin-node#1398]
* Operating System version: N/A
  * Firebase SDK version: 8.3.1
  * Firebase Product: Firestore
  * Node.js version: 11
  * NPM version: 1.0.9

Timestamp.now().seconds is broken with following error:

TypeError: Cannot read property 'seconds' of undefined
    at Function.fromProto (/workspace/node_modules/@google-cloud/firestore/build/src/timestamp.js:136:47)
    at Firestore.snapshot_ (/workspace/node_modules/@google-cloud/firestore/build/src/index.js:715:57)
    at beforeSnapshotConstructor (/workspace/node_modules/firebase-functions/lib/providers/firestore.js:139:30)
    at changeConstructor (/workspace/node_modules/firebase-functions/lib/providers/firestore.js:143:49)
    at cloudFunction (/workspace/node_modules/firebase-functions/lib/cloud-functions.js:133:34)
    at Promise.resolve.then (/layers/google.nodejs.functions-framework/functions-framework/node_modules/@google-cloud/functions-framework/build/src/invoker.js:199:28)
    at process._tickCallback (internal/process/next_tick.js:68:7)

**Comment by MieleVL** [comment]
I'm also seeing this error message out of the blue in Cloud Functions. It's been a couple of months since those functions were updated. Not sure if it's related:

```
TypeError: Cannot read property 'seconds' of undefined
at Function.fromProto (/workspace/node_modules/@google-cloud/firestore/build/src/timestamp.js:135)
at Firestore.snapshot_ (/workspace/node_modules/@google-cloud/firestore/build/src/index.js:678)
at beforeSnapshotConstructor (/workspace/node_modules/firebase-functions/lib/providers/firestore.js:139)
at changeConstructor (/workspace/node_modules/firebase-functions/lib/providers/firestore.js:143)
at cloudFunction (/workspace/node_modules/firebase-functions/lib/cloud-functions.js:133)
```

**Comment by schmidt-sebastian** [comment]
See https://github.com/googleapis/nodejs-firestore/issues/1574.

Fix has been released as `v4.14.1`
